client/src/schema/query.ts

```diff
@@ -58,101 +137,62 @@ export class Query<T extends XataRecord, R extends XataRecord = T> implements Ba
     Object.defineProperty(this, 'repository', { enumerable: false });
   }
 
+  getData(): QueryTableData {
+    return this.#data;
+  }
```

Comment:
Add a public method to retrieve them from the ``Repository``

Reply:
`data`/`getData` could confuse our users thinking that it's the way to execute the query.

What about something like "query options/parameters"

Reply:
Done!